ReplayBrowser/Pages/Shared/MainLayout.razor

```diff
@@ -111,7 +111,7 @@
     <div class="container">
         <p>Replay Browser is a project by Simyon and YuNii. Source code available on <a href="https://github.com/Simyon264/ReplayBrowser">GitHub</a>.</p>
```

Comment:
You could add your name down here, in the future we would probably want a dedicated page for contributors.